rust/src/lib.rs

```diff
@@ -100,4 +100,7 @@ pub enum Error {
     /// JWS signature generation or verification failed.
     #[error("jws signing/verification failed: {0}")]
     JwsError(anyhow::Error),
+    /// Other error.
+    #[error("Error: {0}")]
+    Other(anyhow::Error),
```

Comment:
This is a breaking change unfortunately. IMO we can probably bite the bullet and release a `v0.2.0`, since the churn will be minimal. 

The only alternative I can think of would be to wrap this error in a new type with the new variant, which gets returned by the custom signer, which I don't think will be desirable.